torchci/pages/minihud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -289,14 +447,24 @@ function CommitSummary({ row }: { row: RowData }) {
         row={row}
         numPending={pendingJobs.length}
         showRevert={failedJobs.length !== 0}
+        ttsAlert={ttsAlert}
       />
-      <FailedJobs failedJobs={failedJobs} />
+      {!showDurationInfo && <FailedJobs failedJobs={failedJobs} />}
+      {showDurationInfo && durationJsxElement}
     </div>
   );
 }
 
 function MiniHud({ params }: { params: HudParams }) {
   const data = useHudData(params);
+
+  var paramsNextPage = { ...params };
+  paramsNextPage.page = params.page + 1;
+  const extraRow = useHudData(paramsNextPage);
```

Comment:
why are we fetching the next page? This returns a lot of data.

Reply:
changed the hud api to take a per page parameter, so i can query for 1 commit instead of 50